packages/components-date/src/FieldDateRange/FieldDateRange.stories.tsx

```diff
@@ -43,9 +47,14 @@ export default {
   title: 'Date / FieldDateRange',
 }
 
-const Template: Story<FieldInputDateRangeProps> = (args) => (
-  <FieldDateRange {...args} />
-)
+const Template: Story<FieldInputDateRangeProps & { externalLabel: boolean }> =
+  ({ externalLabel, ...args }) => (
```

Comment:
Set the default for `externalLabel` to true here, then add one or two stories that explicitly have `externalLabel: false`. That way the existing snapshots don't change but we have some new ones to capture the floating label style.

```suggestion
  ({ externalLabel = true, ...args }) => (
```